mvpa2/algorithms/hyperalignment.py

```diff
@@ -179,6 +186,7 @@ class Hyperalignment(ClassWithCollections):
     def __init__(self, **kwargs):
         ClassWithCollections.__init__(self, **kwargs)
         self.commonspace = None
+        self.svm = None
```

Comment:
`svm` is a really confusing name to use here ;) moreover would be nice to have a comment what it actually is here